### ISSUE TITLE
Problem

### ISSUE THREAD

**Issue by merryweatherhq** [XtramCZ/auto-advertise-py#14]
I have been getting this error for a while now.
![image](https://github.com/XtramCZ/auto-advertise-py/assets/159193737/6e833ddd-822d-4784-8a0a-7579b25601aa)

**Comment by merryweatherhq** [comment]
Everything was going good untill today, ive been getting it for a whole day

**Comment by XtramCZ** [comment]
You probably got banned from the server you are sending the message to

**Comment by Zeolite2** [comment]
@merryweatherhq Hey bro, may I ask you how did you start the bot, because everytime I try, I put the token and everything in the  config file, but after that it says "invalid token". Any idea?

**Comment by merryweatherhq** [comment]
you have to place a token from a discord account ur using

**Comment by merryweatherhq** [comment]
@Zeolite2

**Comment by merryweatherhq** [comment]
@Zeolite2 you are supposed to make a discord account not a discord bot

**Comment by Zeolite2** [comment]
yeah, I get the token from a discord account on the Chrome browser, what am I doing wrong?

**Comment by merryweatherhq** [comment]
@Zeolite2 friend me on discord, ill help you there, cba to log in this gitbuh everytime, my discord is "majericc"

**Comment by Zeolite2** [comment]
thank you so much bro, really appreciate it!

**Comment by Zeolite2** [comment]
it says you can't accept friend request, add me "kostura"
![image](https://github.com/XtramCZ/auto-advertise-py/assets/159666679/a1493efe-1291-4928-b6fd-fb0c80a36ebf)

**Comment by Zeolite2** [comment]
@merryweatherhq Hey, can you accept my friend request in Discord, my nickname is "kostura".
